python/tox.ini

```diff
@@ -17,7 +17,7 @@ per-file-ignores =
     # Ignore docstring errors in tests
     *tests/*:D100,D101,D102,D104,D200,ANN101,ANN201,W293,W291,ANN001,E731
     *setup.py:D100
-    *src/aac/templates/*/__init__.py:D104
+    *src/aac/plugins/**/templates/**:D104
```

Comment:
This line may not be adequate if we don't maintain the pattern, like with GenProtobuf.